### PR TITLE
Misnamed arguments to subjects endpoint

### PR DESCRIPTION
**Pull request by eccles**
Problem:
Incorrectly named as plurali.

Solution:
Change wallet_pub_keys to wallet_pub_key
Change tessera_pub_keys to tessera_pub_key

Signed-off-by: Paul Hewlett <phewlett76@gmail.com>

### CODE REVIEW

File: docs/runner/components/subjects_delete.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.. _subjects_delete_yamlref:
```
Comment (henry739) [inline docs/runner/components/subjects_delete.rst:1]: This seems entirely unrelated to the purpose of the PR - Is this here by mistake? If not, please can you mention it in the description.

Comment (eccles) [reply docs/runner/components/subjects_delete.rst:1]: I have added to commit message

**Review by henry739 — APPROVED** [review]
